Documentation/Troubleshooting/ceph-common-issues.md

```diff
@@ -507,6 +507,65 @@ You don't need to restart the pod, the effect will be immediate.
 
 To disable the logging on file, simply set `log_to_file` to `false`.
 
+## Collect coredump and perf data of a ceph process at runtime
+
+There are some cases where the debug logs are not sufficient to investigate issues
+like high CPU utilization of a Ceph process. In that situation, coredump and perf information
+of a Ceph process is useful to be collected which can be shared with the Ceph team in an issue.
+
+To collect this information, please follow these steps:
+
+* Edit the rook-ceph-operator deployment and set `ROOK_HOSTPATH_REQUIRES_PRIVILEGED` to `true`.
+* Wait for the pods to get reinitialized:
+
+```console
+# watch kubectl -n rook-ceph get pods
+```
+
+* Enter the respective pod of the Ceph process which needs to be investigated. For example:
+
+```console
+# kubectl -n rook-ceph exec -it deploy/rook-ceph-mon-a -- bash
+```
+
+* Install `gdb` , `perf` and `git` inside the pod.
```

Comment:
Example command for installing?

Reply:
sure I'll add the example command: `dnf install gdb perf git -y` in there